README.md

```diff
@@ -1,15 +1,29 @@
-# lilly
-Custom Neovim replacement
+<div align="center">
+  <img src="docs/lilly-banner.png" width="445.4" alt="Lilly">
+</div>
 
+## A VIM-Like editor for your terminal (<a href="https://discord.gg/N4UG2TfDfd">chat on Discord</a>)
 ![Screenshot 2023-10-18 00 14 02](https://github.com/tauraamui/lilly/assets/3159648/7d585274-1ec4-42fe-b213-9421aedd4771)
 
-# What?
 
-This project is the home of a custom TUI based text editor. This is intended to replace Neovim as my daily driver once it is sufficiently feature complete.
+Our project is focused on the development of a text editor that serves as a practical alternative to Vim and Neovim. The primary aim is to provide users with essential features, eliminating the need to navigate a complex ecosystem of Lua plugins. This approach is intended to be welcoming to users of all experience levels.
+
+# The Rationale
+### Inclusive Functionality
+We have set out to create an editor that encompasses the fundamental capabilities expected by users, rendering it a compelling choice as a Vim/Neovim alternative. Our emphasis is on streamlining the editing process without the necessity of configuring numerous plugins – our core features aim to fulfill these needs.
+
+### Simplified User Experience
+The intricacies of Lua plugins can be daunting for newcomers and even pose a management challenge for seasoned users. Our editor simplifies the user experience by removing the requirement for extensive plugin management. It offers an approachable and intuitive platform, eliminating the complexities that often accompany plugin management.
+
+### Performance Enhancement
+Our editor is optimized to offer improved performance, particularly when handling extensive files. It is engineered for speed and responsiveness, designed to enhance your editing efficiency.
+
+### VIM-Like Experience
+We've crafted the editor to deliver a VIM-like experience, preserving the functionalities that VIM users appreciate. The difference lies in the absence of reliance on a multifaceted ecosystem – our editor consolidates these features within a unified framework.
+
+Transition to our text editor to explore an alternative that is rooted in functionality, accessibility, and performance.
+
+### Not convinced?
+
```

Comment:
This is the only section with a line break between header and content - it doesn't change the markdown AFAIK, just an observation!

Reply:
True, but it looks nicer to reason about whilst editing.